src/common.tests/Kubernetes/KubernetesClientTests.cs

```diff
@@ -95,7 +97,8 @@ public async Task ListPodsInNamespaceTest()
                                                                             () => _fakeRestClient.CoreV1.ListNamespacedPodWithHttpMessagesAsync(
                                                                                             A<string>._, A<bool?>._, A<string>._, A<string>._,
                                                                                             A<string>._, A<int?>._, A<string>._, A<string>._,
-                                                                                            A<int?>._, A<bool?>._, A<bool?>._, A<IReadOnlyDictionary<string, IReadOnlyList<String>>>._,
+                                                                                            A<bool?>._, A<int?>._, A<bool?>._, A<bool?>._, 

```

Comment:
These changes are due to upgrading KubernetesClient dependency to latest version.

Reply:
Users confirmed it is working. https://github.com/Azure/Bridge-To-Kubernetes/issues/246#issuecomment-1535612103